_posts/2017-10-02-learning-from-8thlight.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title:  "Learning from 8th Light, Day 1"
+date: 2017-10-02 10:03:00 +0200
+categories: how-we-learn 
+author_name: Wolfram Kriesing
+author_url : /author/wolframkriesing
+author_avatar: wolframkriesing
+show_avatar : true
+read_time : 10
+show_related_posts: false
+square_related: recommend-wolf
+feature_image: posts/2017-10-02-8thlight-1/8thlight.jpg
+---
+
+It all started at [BusConf], this August. Tobias Pflug (a colleague from HolidayCheck) asked [Daniel Irvine] (from [8th Light London]) if we could come over to London to learn from 8th Light about their apprenticeship program.
+Why? Because we want to be attractive to talent to stay competitive. And therefore we know we need to improve our skills continuously and keep producing high quality.
+Very simple :).
+
+[Daniel Irvine]: https://twitter.com/d_ir
+[BusConf]: http://www.bus-conf.org
+[8th Light London]: https://8thlight.com/locations/london/
+
+## Boarding the 8th Light plane
+
+So Tobias and me we boarded to London last Tuesday, to spend at least three full days with the 8th Lighters.
+On Wednesday we met at 8:30 in front of the hotel, had a typical english breakfast and went to the 8th Light office. Since it was a nice 20 minutes walk from the hotel to the office, we got to smell some London flair. I always liked London.
+
+Arriving at the office. It was a bit hidden in a side road. We had been very warmly welcomed by Katerina, Gabi, Matt, Mollie, Jim, Daniel, Sarah, Enrique and Rabea. We shook everyone’s hand introduced ourselves, had been shown around and were told to grab any seat we like. In the 8th Light office you find your desk via hot desking, which means you choose to sit any of the available desks. Most of them have a monitor and two keyboards, more about that later.
+Everyone who came later went around and either shook everyone’s hand or fist-bumped with each other in the office. Later Amelia came with Noa, and there the office began lightning up even more when the 9 months old crawled around the office. So cute.
```

Comment:
Isn't it slightly irritating to refer to so many names that nobody can put any faces to?

Reply:
"there the office began" -  `there` refers to a place not a point in time.